tests/config/pubsub/tests.yml

```diff
@@ -1,4 +1,5 @@
-# Supported operation: publish, subscribe
+# Supported operation: publish, subscribe, multiplehandlers, bulkpublish
+# bulkpublish is only run for components that implement pubsub.BulkPublisher interface
```

Comment:
don't you need to update this file and set `allOperations: false` and list the explicit operations everywhere where you don't have bulk publish?

Reply:
It's not clear how `bulkpublish` is added to the list of operations run by `allOperations: true`. Additionally, all operations do need to be listed explicitly where this test should not be run.

Reply:
I agree it's not clear.... 
What I am doing now is that I am skipping the tests for those components that are not implementing this interface.... I think this can lead to cases where silently tests are skipped and not run.... 
I will instead change to scenario where when this operation is specified and the component does not implement the interface,the conf test actually fails.... 

Thanks for noticing this @berndverst

Reply:
Done ....